### PR TITLE
Fix-businesstype-ish

### PR DESCRIPTION
**Pull request by darmhoo**

### CODE REVIEW

File: src/components/PaymentLog.vue [hunk 1]
```diff
@@ -670,7 +672,7 @@ export default {
                 );
                 this.businessTypes = fetchBusinessTypes.data.data.data;
                 this.businessTypes = this.businessTypes.filter(item => {
-                  return item.name.includes("Altara Credit")
+                  return item.slug.includes("ac")
```
Comment (teebarg) [inline src/components/PaymentLog.vue:675]: why not use `ac_`?

File: src/components/PaymentLog.vue [hunk 2]
```diff
@@ -709,7 +711,7 @@ export default {
         const fetchBusinessTypes = await get(this.apiUrls.businessTypes);
         this.businessTypes = fetchBusinessTypes.data.data.data;
         this.businessTypes = this.businessTypes.filter((item) => {
-          return item.name.includes("Altara Credit");
+          return item.slug.includes("ac");
```
Comment (teebarg) [inline src/components/PaymentLog.vue:714]: same here

Comment (darmhoo) [reply src/components/PaymentLog.vue]: done

File: src/components/PaymentLog.vue (hunk 1 above)
Comment (darmhoo) [reply src/components/PaymentLog.vue]: done

**Review by teebarg — APPROVED** [review]

**Pull request merged**